src/main/resources/META-INF/plugin.xml

```diff
@@ -5,7 +5,7 @@
             url="https://research.cs.aalto.fi/LeTech/">LeTech Group at Aalto University
     </vendor>
     <!--  this is the range of supported IntelliJ IDEA version the plugin can be used with  -->
-    <idea-version since-build="213.*"/>
+    <idea-version since-build="213.5744.223"/>
```

Comment:
Hm... It might be that the asterisk is interpreted to mean the latest available version. [The documentation](https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html) ain't very clear on that. Perhaps plain `"213"` could be used to indicate that any 2021.3 version goes. But it doesn't matter in our case, anyway.